crates/gen/src/types/interface.rs

```diff
@@ -135,101 +138,3 @@ impl Interface {
         }
     }
 }
-
-#[cfg(test)]
```

Comment:
Removed internal tests as they required extra scaffolding that isn't needed otherwise and the external tests now more than cover this code.

Reply:
Hmm whole I agree that this code will be exercised in integration tests, having unit tests makes debugging issues easier because it's usually quicker to spot the issue. I think these tests are pretty helpful.